functional-tests/devnet-retrieval

```diff
@@ -125,19 +123,13 @@ echo "miner node starts mining..."
 ./go-filecoin mining start \
   --repodir="$MN_REPO_DIR"
 
-echo ""
```

Comment:
keep an eye on nightly

Reply:
I think this script is going to be deleted. No one is running it and its role will be likely be replaced by something FAST-based.